src/pilot/assign_role.py

```diff
@@ -850,6 +850,16 @@ def assign_role(ip_mac_service_tag, node_uuid, role_index, os_volume_size_gb,
               'path': '/properties/capabilities'}]
     ironic_client.node.update(node_uuid, patch)
 
+    # Detecting BOSS Card and find the boss device size
+    lst_ctrls = drac_client.list_raid_controllers()
+    boss_disk = [ctrl.id for ctrl in lst_ctrls if ctrl.model == "BOSS-S1"]
```

Comment:
Can we change the above to something like cntrl.model.startswith("BOSS")?
This will make it work for versions other than just S1.

Reply:
> Preview
Sure Chis.